ui/components/Navigator.js

```diff
@@ -111,6 +112,13 @@ const styles = (theme) => ({
     display: "inline-table",
     paddingRight: theme.spacing(0.5),
   },
+  listIcon1: {
```

Comment:
@Sudhish2101 thank you. Will you reduce the size of this icon to diminish the amount of attention that it attracts?

Reply:
> Much appreciated, @Sudhish2101. Will you investigate the hiding of these icons on collapse of the drawer?
> <img alt="Screen Shot 2020-11-11 at 7 59 07 PM" width="77" src="https://user-images.githubusercontent.com/7570704/98885889-7abf2000-2458-11eb-88b7-2ccca184b09d.png">

It's the same as before. Only the navigation menu items are visible on collapse of the drawer.